src/test/logging_tests.cpp

```diff
@@ -111,7 +111,7 @@ BOOST_FIXTURE_TEST_CASE(logging_LogPrintf_, LogSetup)
 BOOST_FIXTURE_TEST_CASE(logging_LogPrintMacrosDeprecated, LogSetup)
 {
     LogPrintf("foo5: %s\n", "bar5");
-    LogPrint(BCLog::NET, "foo6: %s\n", "bar6");
+    LogDebug(BCLog::NET, "foo6: %s\n", "bar6");
```

Comment:
This should not be in `logging_LogPrintMacrosDeprecated`, and since it's already covered in the next test `logging_LogPrintMacrosDeprecated` I think this line can simply be removed in the second commit.